subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -309,9 +354,7 @@ private static List<? extends Throwable> getCauses(Throwable cause) {
     }
 
     private void addBuildScanMessage(ContextImpl context) {
-        context.appendResolution(output -> {
-            runtWithOption(output, LONG_OPTION, " to get full insights.");
-        });
+        context.appendResolution(output -> runtWithOption(output, LONG_OPTION, " to get full insights."));
```

Comment:
💭 Not sure if it's appropriate to change in this PR, but I really dislike the name of `runtWithOption`, it seems like a typo of `runWithOption`.

Reply:
Yeah, that seems like a typo: https://github.com/gradle/gradle/pull/28476